modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/CurrencySnapshotAcceptanceManager.scala

```diff
@@ -44,21 +47,29 @@ object CurrencySnapshotAcceptanceManager {
 
     def accept(
       blocksForAcceptance: List[Signed[CurrencyBlock]],
-      lastSnapshotContext: CurrencySnapshotInfo,
+      lastSnapshotContext: (Address, CurrencySnapshotInfo),
       lastActiveTips: SortedSet[ActiveTip],
       lastDeprecatedTips: SortedSet[DeprecatedTip],
       calculateRewardsFn: SortedSet[Signed[CurrencyTransaction]] => F[SortedSet[RewardTransaction]]
     ) = for {
-      acceptanceResult <- acceptBlocks(blocksForAcceptance, lastSnapshotContext, lastActiveTips, lastDeprecatedTips)
+      currencyIdentifier <- lastSnapshotContext.address.value.value.hashF
```

Comment:
I would find a different name for it to not logically mix it with the metagraph/currency identifier being the address. At this point it's the `initalTxHash` or something like that.

Reply:
done :+1: